### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by nikkypyra**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -8,56 +8,116 @@ import PetForm from './pages/PetForm'
...
+      setTasks(allTasks)
+    })
+  }, [])
+
+  const [previewImage, setPreviewImage] = useState({
+    imageUrl:
+      'https://firebasestorage.googleapis.com/v0/b/pawlog-app.appspot.com/o/images%2Ftaskpaw.png?alt=media&token=8ad10974-93e4-4fd7-ae05-1567d049ad1f',
+    imageName: 'taskpaw.png',
+  })
+
```
Comment (juliaknuth) [inline src/App.js:41]: maybe you can outsource your whole image upload to its own component

File: src/components/Navigation.js [hunk 2]
```diff
@@ -4,13 +4,13 @@ import { NavLink } from 'react-router-dom'
 import { ReactComponent as Home } from '../icons/home.svg'
 import { ReactComponent as Filter } from '../icons/filter.svg'
 
-export default function Footer() {
+export default function Navigation() {
   return (
     <FooterStyled>
-      <NavStyled exact to="/" activeClassName="chosen">
+      <NavStyled exact to="/" activeClassName="chosen" data-cy="home">
```
Comment (juliaknuth) [inline src/components/Navigation.js:10]: you can style your active navigation element inline with activeStyle="color", no more classes needed

**Review by juliaknuth — COMMENTED** [review]
Looks good :)

File: src/App.js (hunk 1 above)
Comment (nikkypyra) [reply src/App.js:41]: great idea!

File: src/components/Navigation.js (hunk 2 above)
Comment (nikkypyra) [reply src/components/Navigation.js:10]: good tip, thank you!

**Pull request merged**